Add tunnelName and tunnelOwner to test configuration options (#1414)
saucelabs/sauce-docs: docs/dev/test-configuration-options.md

"public": "team"
```

---
### `tunnelName`
<p><small>| STRING |</small></p>

> Specify a [Sauce Connect](/secure-connections/sauce-connect) tunnel to establish connectivity with a Sauce Labs test platform. Tunnels allow you to test an application that is behind a firewall or on your local machine by providing a secure connection to the Sauce Labs platform.

Reviewer: ```suggestion
Specify a [Sauce Connect tunnel name](/secure-connections/sauce-connect/setup-configuration/basic-setup/#using-tunnel-names) to establish connectivity with a Sauce Labs test platform.
```

:::note Choose the Correct Tunnel Identifier
The value expected here is the value shown under the **Tunnel Name** column on the Sauce Labs Tunnels page, _not_ the **Tunnel ID** numerical value.
:::

See [Using Tunnel Names](/secure-connections/sauce-connect/setup-configuration/basic-setup/#using-tunnel-names) for more information.

```java
"tunnelName": "MyTunnel01"
```

---
### `tunnelIdentifier`
<p><small>| STRING |</small></p>

:::caution Deprecation notice
`tunnelIdentifier` is being deprecated in favor of `tunnelName`.
:::

> Specify a [Sauce Connect](/secure-connections/sauce-connect) tunnel to establish connectivity with a Sauce Labs test platform. This is alias for [tunnelName](#tunnelname).

Reviewer: ```suggestion
Specify a [Sauce Connect tunnel name](/secure-connections/sauce-connect/setup-configuration/basic-setup/#using-tunnel-names) to establish connectivity with a Sauce Labs test platform. This is alias for [tunnelName](#tunnelname).
```

"tunnelIdentifier": "MyTunnel01"
```

:::warning Breaking Change

> Appium tests for the Real Device Cloud using the W3C protocol MUST use `tunnelName` instead of `tunnelIdentifier`.

Reviewer: is that so? I think they support either tunnelName or tunnelIdentifier.

:::

---
### `tunnelOwner`
<p><small>| STRING |</small></p>

If the [tunnelName](#tunnelname) (or [tunnelIdentifier](#tunnelidentifier)) you've specified to establish connectivity with a Sauce Labs test platform is a shared tunnel, and you are _not_ the user who created the tunnel, you must identify the Sauce Labs user who did create the tunnel in order to use it for your test.

```java
"tunnelName": "MyTeamSharedTunnel"
"tunnelOwner": "<username of tunnel originator>"
```

---
### `parentTunnel`
<p><small>| STRING |</small></p>

:::caution Deprecation notice
`parentTunnel` is being deprecated in favor of `tunnelOwner`.
:::

If the [tunnelName](#tunnelname) (or [tunnelIdentifier](#tunnelidentifier)) you've specified to establish connectivity with a Sauce Labs test platform is a shared tunnel, and you are _not_ the user who created the tunnel, you must identify the Sauce Labs user who did create the tunnel in order to use it for your test. This is alias for [tunnelOwner](#tunnelowner).

```java
"tunnelName": "MyTeamSharedTunnel"
"parentTunnel": "<username of tunnel originator>"
```

:::warning Breaking Change

> Appium tests for the Real Device Cloud using the W3C protocol MUST use `tunnelOwner` instead of `parentTunnel`.

Reviewer: I'd suggest confirming this
